### ISSUE TITLE
Add spawn objects to fresh install of program?

### ISSUE THREAD

**Issue by lordpigu** [Arkensor/DayZCommunityOfflineMode#271]
Is there anyway to add all my spawn objects to a fresh install of dayzcommunityofflinemode so that when it loads up all my added objects spawn in and I can continue adding to and removing my objects as if using the program normally?

**Comment by lordpigu** [comment]
I still have all my "Spawnobject" objects in a seperate file though. I just need to know if I can put this list of objects into dayzofflinemode so it loads with all my added buildings both on the map and in the list of added items on the bottom right.

**Comment by lordpigu** [comment]
OK i figured out how to load all my spawn objects into my objecteditor.c file HOWEVER when I do all 1000 ish objects the batch crashes just before it finishes loading. Any fix or help with that?

**Comment by lordpigu** [comment]
I can do 5 objects easily and everything works but if I try all of my objects it crashes.

**Comment by lordpigu** [comment]
Ok so I got all my objects spawning into the server copying all my objects into the objecteditor.c file HOWEVER when I edit something and save the program then close out and reload it, nothing changes. I hit save a couple times and now when I load into theres multiples of every object I've place. Was I supposed to add all the objects into the objecteditor.c file, load the game, then save once and then remove the objects i pasted into the objecteditor.c file?

**Comment by lordpigu** [comment]
Ok, so i figured out how to get my objects back into my game. I copied all my "SpawnObjects" into the ObjectEditor.c file and then loaded up dayzcommunityofflinemode. Would crash without setting protected bool HIVE_ENABLED = false. It would then load up with all my objects. I clicked save once then exited the application and went back to ObjectEditor.c and deleted all my "SpawnObjects" as they were now saved when I clicked save and didn't want to load them again and have duplicates everywhere. It worked however a lot of the objects are either moved slightly to one side or the elevation of them is reset. When I fix them and click save it saves their location but not elevation for some reason. Super confused and frustrated. Any help would be appreciated.
